core/state-machine/src/trie_backend.rs

```diff
@@ -147,6 +147,12 @@ impl<S: TrieBackendStorage<H>, H: Hasher> Backend<H> for TrieBackend<S, H> where
 		(root, write_overlay)
 	}
 
+	fn full_storage_root<I>(&self, delta: I) -> (H::Out, Self::Transaction)
+		where I: IntoIterator<Item=(Vec<u8>, Option<Vec<u8>>)>
+	{
+		self.storage_root(delta)
```

Comment:
Probably want this to be `unimplemented!` first as it clearly doesn't give expected values.